actions/kpops-runner/README.md

```diff
@@ -0,0 +1,29 @@
+# kpops-runner
+
+This action runs Kpops with the given config.
+
+## Input Parameters
+
+| Name              | Required | Default Value |                                                     Type                                                      | Description                                                                     |
+| ----------------- | :------: | :-----------: | :-----------------------------------------------------------------------------------------------------------: | ------------------------------------------------------------------------------- |
+| mode              |    ✅    |       -       | 													string  										           | command mode used by Kpops (deploy, destroy, reset, clean)) for possible values |
+| pipeline          |    ✅    |       -       |                                                    string                                                     | Pipeline file to be run by Kpops publication                                    |
+| working-directory |    ❌    |       .       |                                                    string                                                     | root directory used by Kpops to run pipelines                                   |
+| execute           |    ❌    |  	false     |												 		string	  													| Execute Kpops command (this applies the infrastructure changes that were executed inside the dry-run command) |
+| defaults          |    ❌    |    defaults   | 													string 														|  defaults folder path 														|
+| config          |    ❌    |    config.yaml   | 													string 														|  default config.yaml file path 														|
```

Comment:
```suggestion
| config          |    ❌    |    config.yaml   | 													string 														|  config.yaml file path 														|
```

Reply:
Is there a reason you want to keep the `default` here since you resolved the thread?

Reply:
no, it was my fault. Now should be fixed :)